### ISSUE TITLE
Combine leaves small unworked pieces on turns

### ISSUE THREAD

**Issue by iljarogoff** [Courseplay/courseplay#3817]
Happens only on JD T560 with 7.6m header, wide headers have been successfully fixed already.

![fsScreen_2019_05_20_21_08_21](https://user-images.githubusercontent.com/21023840/58045871-0c8bec80-7b44-11e9-9f99-7960c57b044b.png)
![fsScreen_2019_05_20_21_08_37](https://user-images.githubusercontent.com/21023840/58045872-0dbd1980-7b44-11e9-9375-683f0ea64102.png)
![fsScreen_2019_05_18_17_53_22](https://user-images.githubusercontent.com/21023840/58045878-1281cd80-7b44-11e9-94eb-d6787944c148.png)

**Comment by Tensuko** [comment]
Does this happen, when you add 1 Headland to the fieldwork ?

**Comment by iljarogoff** [comment]
Unfortunately yes. Not so often, but still happens. Just some fine tuning, nothing to worry about. Next time working with the T560 I'm going to set the work width to 7.5m instead of 7.6m, maybe this will fix it.
![fsScreen_2019_05_22_06_16_36](https://user-images.githubusercontent.com/21023840/58147156-806be900-7c59-11e9-851f-b8c2f1999482.png)

**Comment by RUGhost** [comment]
if you set turning circle 15m also eliminate this problem. But a fine tuning will be nice.

**Comment by iljarogoff** [comment]
Not really. Tried this and it still leaves patches.
![fsScreen_2019_05_22_16_38_13](https://user-images.githubusercontent.com/21023840/58183766-2b0ef680-7cb0-11e9-8ba9-a836dc73429d.png)

**Comment by pvaiko** [comment]
I can't reproduce this on Ravenport Field 8. @Tensuko , you want to try?

**Comment by Tensuko** [comment]
I will do some test with different Combines and Cutters.
@pvaiko need any debug chanel and screenshot ?

**Comment by pvaiko** [comment]
channel 12 and a screenshot to see the turn maneuver it is doing.

**Comment by Tensuko** [comment]
Tested with:
13.5m
12m
9m
7.6m
5m
(and the smalest one, forgot the meters...)

What I tried just now again, was your Vehicle and the result was, that sometimes, it leaves a very smal piece, yes. But then I return into the field before that turn (far enough so the Combine stays straight on the Line) and let him do the same line again and he takes away the fruit before the turn. So I say this is very random that this happen, where we hardly can do anything. Reducing the work width should help.

Got to close this, as we can't do much about that, sry.
